src/scenes/title.js

```diff
@@ -124,7 +122,7 @@ class TitleScreen extends Phaser.Scene {
 
 
 	  	//Gets user input to change the scene to the game
-	  	this.input.on('pointerdown', () => this.scene.start('GameScreen'))
+	  	this.input.on('pointerdown', () => this.scene.start('SelectionScreen'))
```

Comment:
Just changed the title screen to go to the character select screen rather than the main game